src/BasketBundle/Controller/Api/BasketController.php

```diff
@@ -81,21 +81,35 @@ public function __construct(BasketManagerInterface $basketManager, BasketElement
      *
      * @ApiDoc(
      *  resource=true,
-     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"="sonata_api_read"}
+     *  output={"class"="Sonata\DatagridBundle\Pager\PagerInterface", "groups"={"sonata_api_read"}}
      * )
      *
      * @QueryParam(name="page", requirements="\d+", default="1", description="Page for baskets list pagination (1-indexed)")
      * @QueryParam(name="count", requirements="\d+", default="10", description="Number of baskets by page")
-     * @QueryParam(name="orderBy", array=true, requirements="ASC|DESC", nullable=true, strict=true, description="Query baskets basket by clause (key is field, value is direction")
      *
-     * @View(serializerGroups="sonata_api_read", serializerEnableMaxDepthChecks=true)
+     * @View(serializerGroups="{sonata_api_read"}, serializerEnableMaxDepthChecks=true)
```

Comment:
Careful, it should be: {"sonata_api_read"} instead of "{sonata_api_read"}

Reply:
it happens everywhere I believe

Reply:
Ah, dammit. Yeah, I just mass-replaced everything...